### PR TITLE
[PPANTT-135] chore: defined re schema

### PR DESCRIPTION
**Pull request by gioelemella**
#### List of Changes

- defined payment-options-re json schema

#### Motivation and Context
[PPANTT-135](https://pagopa.atlassian.net/browse/PPANTT-135)

#### Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

#### Checklist:

- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.

[PPANTT-135]: https://pagopa.atlassian.net/browse/PPANTT-135?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

**Review by svariant — APPROVED** [review]

**Review by jacopocarlini — APPROVED** [review]

File: templates/payment-options-re.json [hunk 1]
```diff
@@ -0,0 +1,105 @@
...
+          "description": "Data inserimento evento su kafka",
+          "examples": ["2024-09-23T17:27:04.1696470"]
+        },
+        "eventType": {
+          "type": "string",
+          "enum": ["REQ","RES","INTERNAL"]
+        },
+        "organizationId": {
+          "type": "string",
+          "description": "Id dell’EC"
```
Comment (pasqualespica) [inline templates/payment-options-re.json:23]: force len to `11` https://json-schema.org/understanding-json-schema/reference/string#length

File: templates/payment-options-re.json [hunk 2]
```diff
@@ -0,0 +1,105 @@
...
+        },
+        "stationId": {
+          "type": "string",
+          "description": "Id della stazione"
+        },
+        "pspId": {
+          "type": "string",
+          "description": "Id del PSP"
```
Comment (pasqualespica) [inline templates/payment-options-re.json:31]: ditto as for `organizationId`

File: templates/payment-options-re.json [hunk 3]
```diff
@@ -0,0 +1,105 @@
...
+        },
+        "brokerId": {
+          "type": "string",
+          "description": "Id del broker"
+        },
+        "iuv": {
```
Comment (pasqualespica) [inline templates/payment-options-re.json:37]: for iuv and notice could be use https://json-schema.org/understanding-json-schema/reference/regular_expressions ? right @jacopocarlini @aomegax @marilenapaldino87 ?

File: templates/payment-options-re.json [hunk 4]
```diff
@@ -0,0 +1,105 @@
...
+          "type": "string"
+        },
+        "status": {
+          "type": "string",
+          "enum": ["OK","KO"]
+        },
+        "errorStatusDesc": {
+          "type": "string"
+        },
+        "errorStatusCode": {
```
Comment (pasqualespica) [inline templates/payment-options-re.json:50]: it its our ODP-<error> use use regex- livk "ODP-" + 4 digits example
and add link to ODP open-api as legend
https://json-schema.org/learn/miscellaneous-examples#regular-expression-pattern

File: templates/payment-options-re.json [hunk 5]
```diff
@@ -0,0 +1,105 @@
...
+          "type": "string"
+        },
+        "sessionId": {
+          "type": "string"
+        },
+        "payload": {
+          "type": "string"
```
Comment (pasqualespica) [inline templates/payment-options-re.json:57]: add description to specify encoding ( base64, gzip , ... )

File: templates/payment-options-re.json [hunk 6]
```diff
@@ -0,0 +1,105 @@
...
+        },
+        "eventTimestamp": {
+          "type": "string",
+          "description": "Data e ora dell’evento",
+          "examples": ["2024-09-23T17:27:04.1690000"]
+        },
+        "version": {
```
Comment (pasqualespica) [inline templates/payment-options-re.json:64]: number `1`, ...

File: templates/payment-options-re.json [hunk 7]
```diff
@@ -0,0 +1,105 @@
...
+  "properties": {
+    "body": {
+      "type": "object",
+      "properties": {
+        "id": {
+          "type": "string"
+        },
+        "insertedTimestamp": {
+          "type": "string",
+          "description": "Data inserimento evento su kafka",
```
Comment (pasqualespica) [inline templates/payment-options-re.json:14]: into to descriptio specify datatime format type

**Review by pasqualespica — CHANGES_REQUESTED** [review]
lgtm 🆗 
evaluate some change proposal

**Pull request merged**